_layouts/hub_detail.html

```diff
@@ -20,14 +20,19 @@ <h1>
         </h1>
 
         <div class="row">
-          <div class="col-md-6">
+          <div class="col-md-4">
             <p class="detail-lead">By {{ page.author }} </p>
           </div>
 
-          <div class="col-md-6">
+          <div class="col-md-8">
             <p class="detail-lead lead-summary">{{ page.summary }}</p>
-            <a href="{{ page.github-link }}"><button class="btn btn-lg with-right-white-arrow detail-github-link">View on Github</button></a>
-            <a href="https://colab.research.google.com/github/pytorch/pytorch.github.io/blob/master/{{ page.path | replace: "_hub", "assets/hub" | replace: ".md", ".ipynb" }}"><button class="btn btn-lg with-right-white-arrow detail-colab-link">Open on Google Colab</button></a>
+            <div class="detail-button-contain">
```

Comment:
nit - can you change the class name to detail-button-container?

Reply:
Sure thing, I just updated the class name